### PR TITLE
UI deployment - Molecule tests

### PR DESCRIPTION
**Pull request by rzadp**
- Renamed existing scenario `default` to `oracle`
- Introduced `ui` scenario
- `prepare.yml` shared between both scenarios
- `test_all.py` cases shared between both scenarios
- Closes https://github.com/poanetwork/tokenbridge/issues/129

### CODE REVIEW

File: deployment/molecule/ui/tests/test_ui.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
```
Comment (akolotov) [inline deployment/molecule/ui/tests/test_ui.py]: which test here does check that the UI is running - the server responds with a correct content?

Comment (rzadp) [reply deployment/molecule/ui/tests/test_ui.py]: I've added two tests:
- Check that the server responds with html with proper title
- Checks that returned html does not contain `error` string (with case insensitivity)

Comment (akolotov) [reply deployment/molecule/ui/tests/test_ui.py]: thanks!

**Comment by akolotov** [comment]
Now we can merge it

**Pull request merged**